pyxdf/setup.py

```diff
@@ -0,0 +1,11 @@
+from setuptools import setup
+
+setup(name='pyxdf',
+      version='0.1',
+      description='Python library for importing XDF (Extensible Data Format)',
+      url='http://github.com/sccn/xdf',
+      author='Chadwick Boulay',
```

Comment:
Should be `Christian Kothe` (for setup.py, only one author should be specified so it could also be `Alejandro Ojeda`)

Reply:
I'd try to contact the original authors Christian Kothe and Alejandro Ojeda - if they don't want to maintain the package, I'd prefer to specify the current maintainer's contact details in `setup.py`. I think this field is supposed to contain the package author anyway, but I'm not sure.